services/brig/src/Brig/Run.hs

```diff
@@ -56,9 +60,13 @@ run o = do
     Async.async
       $ AWS.execute (e ^. awsEnv)
       $ AWS.listen throttleMillis q (runAppT e . SesNotification.onEvent)
+  sftDiscovery <- case e ^. sftEnv of
+    Nothing -> Async.async (pure ()) -- TODO: This looks fishy
```

Comment:
it's for backends that don't want to do calling.  maybe we should just handle this differently in the calling config end-points as well?  like, return 404 instead of an empty sft server list?  or should we not allow this case?  force people to run wire with calling always?

Reply:
We could make `sftDiscovery` a `Maybe` and `mapM_` into it for cancelling.  That would be slightly nicer; no lingering thread.

Reply:
Looks much better like this, thanks!